### DISCUSSION TITLE
Давайте где-нибудь постить работающие командные строки для разных операторов?

### DISCUSSION THREAD

**Discussion by denizzzka** [hufrea/byedpi#184] [General]
Например, долгое время работала такая строчка, но неделю назад ~~перестала~~ начала работать со сбоями:
```
ciadpi --disorder=4 --auto=torst,sid_inv,alert --timeout=2.5 --disorder=7
```

**Comment by sgt-cartman** [comment]
Билайн, провод. Ютуб сегодня перестал грузить видео. Параметры пробовал такие:
ciadpi -s1 -o1 -Ar -o1 -At -f-1 -r1+s -As
ciadpi -s1 -o1 -Ar -o3 -At -f-1 -r1+s -As
ciadpi --oob 1

**Comment by waptaxi** [comment]
> Билайн, провод. Ютуб сегодня перестал грузить видео. 

попробуй `-s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An
`

**Reply by cravvler9** [reply]
> > Билайн, провод. Ютуб сегодня перестал грузить видео.
> 
> попробуй `-s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An `

Ростелеком, заработало, спасибо!

**Reply by madbanana22** [reply]
работает, провайдер игра-сервис, спасибо!

**Reply by shart7** [reply]
> > Билайн, провод. Ютуб сегодня перестал грузить видео.
> 
> попробуй `-s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An `

Спасибо! 
МГТС Москва идеально работает Android, iOS, приставка MiBox; в браузере необходимо отключить поддержку TLS 1.3 hybridized Kyber support

**Reply by VanillaDphdmn** [reply]
> > Билайн, провод. Ютуб сегодня перестал загружать видео.
> 
> попробуй`-s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An `

Заработал дискорд на мобильном интернете. как и другие заблокированные сайты. До этого у меня bydpi на мобильном интернете был бесполезен, прям как и все впны которые я пробовал, не знаю с чем связана эта проблема, но главное что у меня теперь есть дискорд вне дома. Спасибо!

**Reply by Demon-i** [reply]
Странно. Билайн кабель Моск.Обл. Месяц работал без DPI везде. Даже на теликах.. Ща всё сломалось так, что никакие настройки не работают. Может кто нашел что работающее?

**Comment by trevarj** [comment]
https://github.com/hufrea/byedpi/discussions/58

**Comment by sgt-cartman** [comment]
> > Билайн, провод. Ютуб сегодня перестал грузить видео.
> 
> попробуй `-s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An `

Большое спасибо. Так работает.

**Comment by slesru** [comment]
Не , так не работает:

При соединении с www.youtube.com произошла ошибка. Узел сообщает о несовместимой или неподдерживаемой версии протокола.

Код ошибки: SSL_ERROR_PROTOCOL_VERSION_ALERT

**Comment by chemizt** [comment]
Работает для ютуба на московском МГТС с параметрами из [этого комментария](https://github.com/hufrea/byedpi/discussions/58#discussioncomment-10899209). Раздупляется не сразу, но когда раздуплится - видео догружается до конца.

**Comment by EMWD** [comment]
Для ютуба странно себя ведёт программа сегодня с утра, начинает грузить видео только через 15-20 секунд, грузит их непродолжительное время, затем отваливается до перезагрузки. юго-восток москвы, МТС. Пробовал разные комбинации флагов - не помогло.

**Reply by ALV-036** [reply]
До  9 октября  всё летало при строке  ./ciadpi-x86_64 --drop-sack --tfo --md5sig --split 2 --split 3 --tlsrec 1+s --auto=torst --timeout 1

С 10 октября уже не так хорошо,     сейчас попробовал рекомендованную строку из комментариев выше
./ciadpi-x86_64 -s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An           и ютуб снова летает.

Но создаётся впечатление, что провайдеры сидят читают эти форумы и подстраивают свой блокирующий софт именно под эти строки которые мы сообщаем в чате.     Потому что выложенные в чат строки конфигурации живут от силы 1-2 недели.

**Reply by denizzzka** [reply]
> Но создаётся впечатление, что провайдеры

Им оно меньше всего надо. Им централизованно прошивку обновляют в блокировочных фильтрах, над которыми у них нет контроля

**Reply by poiNt3D** [reply]
> С 10 октября уже не так хорошо, сейчас попробовал рекомендованную строку из комментариев выше ./ciadpi-x86_64 -s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An и ютуб снова летает.


Подскажите пожалуйста, почему у меня не признаёт эту строчку? Пишет что нет такой опции -Y
Версия последняя из релизов - 0.14.1
```
ciadpi.exe  -s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An
ciadpi.exe: unknown option -- Y
```

**Reply by dimuls** [reply]
> Пишет что нет такой опции -Y

Винда и линукс поддерживают разный набор параметров. Можно попробовать просто убрать его

**Comment by rukroot** [comment]
У меня просто `ciadpi -o` и разными числами пока на 1 остановился работает, но при этом иногда не открывается интернет и некоторые сайты
В консоли полно `recv: Connection reset by peer`
Кстати хороший способ проверить работает или нет, встроенная стата на ютубе, когда норм, то скорость соеденения огромная 70Мбит и выше, а когда режут то мало, сразу видно
Есть совет что сделать чтобы другие сайты не задевало?
Сейчас он как общесистемный прокси включен в убунте

**Reply by dmitryam** [reply]
флаг -H и список хостов в файле

**Reply by edvejak** [reply]
> У меня просто `ciadpi -o` и разными числами пока на 1 остановился работает, но при этом иногда не открывается интернет и некоторые сайты В консоли полно `recv: Connection reset by peer` Кстати хороший способ проверить работает или нет, встроенная стата на ютубе, когда норм, то скорость соеденения огромная 70Мбит и выше, а когда режут то мало, сразу видно Есть совет что сделать чтобы другие сайты не задевало? Сейчас он как общесистемный прокси включен в убунте

You can install firefox extension: toggle proxy

**Comment by koshcheev713** [comment]
04.11.2024
Какой-то оператор
Заработал старый конфиг: ciadpi --oob 1 --split 2 --mod-http h,d --auto torst --fake -1 --tlsrec 3+s --md5sig --auto none

**Comment by walmstudio** [comment]
Работает как с Ростелеком и Метромакс, так и с телефонными операторами Tele2 и Yota
ciadpi --proto=udp --pf=443 --fake-data=":\xC2\x00\x00\x00\x01\x14\x2E\xE3\xE3\x5F\x6B\xBB\x23\xA8\xE6\x5D\xA9\x78\x21\xCF\xC2\x72\x4C\x8F\xC4\x5E\x14\x00\x00\x00\x00\xC5\x00\x00\x00\x00\x4C\x00\xA7\x00\x00\x00\x00\x00\x00\x44\x00\x00\x80\x00\x00\x00\x0D\xFC\xFA\x1D\xCD\x73\xBA\x2A\x90\x93\xB3\xEE\xF7\x43\xC5\x85\xDA\xFF\x45\x3C\x00\x00\x00\x00\x00\x00\x7C\x00\x9B\x00\xF6\x00\x00\xDD\x00\x00\x00\x00\x00\x00\x00\x00\x00\x59\xA8\xE4\x00\x00\x00\x00\x00\x00\x00\x00\x7B\x00\x0F\x00\x00\x00\x48\x4E\x00\x00\x00\x06\xF3\x00\x00\x00\x00\xD9\x5A\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00" --udp-fake=10 --auto=none -s0 -o1 -d1 -r1+s -Ar -o1 -At -f-1 -r1+s -As

**Reply by Jenstel** [reply]
> Работает как с Ростелеком и Метромакс, так и с телефонными операторами Tele2 и Yota ciadpi --proto=udp --pf=443 --fake-data=":\xC2\x00\x00\x00\x01\x14\x2E\xE3\xE3\x5F\x6B\xBB\x23\xA8\xE6\x5D\xA9\x78\x21\xCF\xC2\x72\x4C\x8F\xC4\x5E\x14\x00\x00\x00\x00\xC5\x00\x00\x00\x00\x4C\x00\xA7\x00\x00\x00\x00\x00\x00\x44\x00\x00\x80\x00\x00\x00\x0D\xFC\xFA\x1D\xCD\x73\xBA\x2A\x90\x93\xB3\xEE\xF7\x43\xC5\x85\xDA\xFF\x45\x3C\x00\x00\x00\x00\x00\x00\x7C\x00\x9B\x00\xF6\x00\x00\xDD\x00\x00\x00\x00\x00\x00\x00\x00\x00\x59\xA8\xE4\x00\x00\x00\x00\x00\x00\x00\x00\x7B\x00\x0F\x00\x00\x00\x48\x4E\x00\x00\x00\x06\xF3\x00\x00\x00\x00\xD9\x5A\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00" --udp-fake=10 --auto=none -s0 -o1 -d1 -r1+s -Ar -o1 -At -f-1 -r1+s -As

Это полный конфиг или отдельная опция для UDP?

**Reply by walmstudio** [reply]
Полный конфиг

**Comment by Mihrutkin** [comment]
господа, помогите. Вот работающая команда -Ku -a3 -An -Kt,h -n [vk.com](http://vk.com/) -o1 -d1 -Mh,d -At,r,s -Kt -n [vk.com](http://vk.com/) -s0+sm -d1 -s5+s -s10+s -s15+s -s20+s -t15 -b4500 -r1+s -S -As -Kt -n [vk.com](http://vk.com/) -s0+sm -d1 -s5+s -s10+s -s15+s -s20+s -t15 -b4500 -S -An

Как  правильно нужно дописать следующую команду, чтобы работало только на сайты из файла  --hosts /etc/config/byedpi.hosts
Спасибо

**Reply by walmstudio** [reply]
Пишешь параметр --hosts=путь/к/файлу.txt

**Reply by Mihrutkin** [reply]
Вот так верно? Я писал про объединить эти две команды в одну

 --hosts /etc/config/byedpi.hosts -Ku -a3 -An -Kt,h -n [vk.com](http://vk.com/) -o1 -d1 -Mh,d -At,r,s -Kt -n [vk.com](http://vk.com/) -s0+sm -d1 -s5+s -s10+s -s15+s -s20+s -t15 -b4500 -r1+s -S -As -Kt -n [vk.com](http://vk.com/) -s0+sm -d1 -s5+s -s10+s -s15+s -s20+s -t15 -b4500 -S -An

**Reply by walmstudio** [reply]
Все правильно

**Comment by Mihrutkin** [comment]
-Ku -a1 -An -o1 -At,r,s -d1 -a2

-o1 -o25+s -T3 -At --tlsrec 1+s -a2

**Reply by Mihrutkin** [reply]
-s1 -q1 -Y -a2 -Ar,t -f-1 -r1+s -a3

**Comment by DroidProger** [comment]
Использую такое на проводном билайне

--hosts /etc/byedpi/yt.txt --oob 1 --tlsrec 5+s --auto=none --hosts /etc/byedpi/ds.txt --oob 2 --tlsrec 3+s --udp-fake 1 --auto=none --split 5:1 --disoob 15 --drop-sack

yt.txt - домены ютуба (youtube.com googlevideo.com youtubemobilesupport.com youtubei.googleapis.com)
ds.txt - домены дискорда (discordsays.com discord.com discord.dev)

Возможно в доменах есть что то лишнее)

**Reply by DroidProger** [reply]
C недавних пор не работают сайты за cloudflare - режется ответ от сервера, страницы не успевают полностью загрузится. Способов исправить пока не найдено :(

**Reply by denizzzka** [reply]
> yt.txt - домены ютуба (youtube.com googlevideo.com youtubemobilesupport.com youtubei.googleapis.com)

Это не все домены, ещё есть:

youtu.be
yt3.ggpht.com
ytimg.com

Если их не добавить то не прогружаются превьюхи и всякое такое

**Comment by sour-source** [comment]
Теле2, Мегафон
Дальний Восток
Собрал вот такой конфиг из разных кусков. Работает всё что мне нужно: Instagram, Youtube, Discord (звонки тоже). Также поднимется Gemini на телефоне если в автоматический режим закинуть приватные днс `dns.comss.one`
Клиент byebyeDPI, идею мультистратегии брал из доков zapret, списки доменов от туда же.
`--ip 127.0.0.1 --port 10080 --hosts /storage/emulated/0/GoodbyeDPI/hostlist-youtube.txt -o1 -s4 -s6 -a1 --auto=none --hosts /storage/emulated/0/GoodbyeDPI/hostlist-discord.txt -n 4pda.to -Qr -f-204 -s1:5+sm -a1 -As -d1 -s3+s -s5+s -q7 -a1 -As -o2 -f-43 -a1 -As -r5 -Mh -s1:5+s -s3:7+sm -a1 --auto=none --hosts /storage/emulated/0/GoodbyeDPI/hostlist-user.txt -s1+s -d3+s -a1`

**Reply by Jenstel** [reply]
Можете, пожалуйста, приложить файлы хост-листов?

**Reply by sour-source** [reply]
> Можете, пожалуйста, приложить файлы хост-листов?

[hostlist-discord.txt](https://github.com/user-attachments/files/23573144/hostlist-discord.txt)
[hostlist-user.txt](https://github.com/user-attachments/files/23573145/hostlist-user.txt)
[hostlist-youtube.txt](https://github.com/user-attachments/files/23573146/hostlist-youtube.txt)

**Comment by Onair-santa** [comment]
-Kt,h -d7 -s2 -An -Ku -a9 -An
Отлично работает долгое время и никаких проблем.

**Reply by DroidProger** [reply]
Неплохо было бы написать какой оператор и регион

**Reply by antonnagorniy** [reply]
Северо-Запад, Ростелеком, Win11 отлично работает

**Comment by vitalfadeev** [comment]
### Что работает ?

Только **Youtube**. Остальное работает как обычно - что-то работает, что-то блокируется. По необходимости включаю **adguard-vpn**.

Использую **ciadpi** и расширение chrome **zeroomega**, вот  такое.
<img width="1339" height="653" alt="image" src="https://github.com/user-attachments/assets/4b80ca57-c8cf-4fd3-b5b1-562c38ece14d" />


### Конфиги

Конфиг **ciadpi** вот такой:
```
-i 127.0.0.1 -p 1080  --timeout 2 --tfo --ttl 6 --round 2
```

Конфиг **zeroomega** вот такой:

```
[SwitchyOmega Conditions]
@with result

*.youtube.com +proxy
*.youtu.be +proxy
*.yt.be +proxy
*.googlevideo.com +proxy
*.ytimg.com +proxy
*.ggpht.com +proxy
*.gvt1.com +proxy
*.youtube-nocookie.com +proxy
*.youtube-ui.l.google.com +proxy
*.youtubeembeddedplayer.googleapis.com +proxy
*.youtube.googleapis.com +proxy
*.youtubei.googleapis.com +proxy
*.yt-video-upload.l.google.com +proxy
*.wide-youtube.l.google.com +proxy
*.fonts.googleapis.com +proxy
*.fonts.gstatic.com +proxy
*.www.gstatic.com +proxy
*.googleads.g.doubleclick.net +proxy
*.static.doubleclick.net +proxy
*.www.google.com +proxy
*.www.google.ru +proxy
*.googleusercontent.com +proxy

* +direct
```

<img width="1339" height="653" alt="image" src="https://github.com/user-attachments/assets/f69beb3f-debe-4305-80dd-805f05971efd" />

и

<img width="1339" height="653" alt="image" src="https://github.com/user-attachments/assets/0dab41a5-1e87-41f0-8b70-cb2e176c556c" />

#### Конфиг системного hosts.

Чтобы youtube не подвисал на шортсах, игноируем бесконечно замедленные IP, конфиг **/etc/hosts** вот такой:

```
127.0.0.1  rr1---sn-gvnuxaxjvh-3w5s.googlevideo.com
127.0.0.1  rr2---sn-gvnuxaxjvh-3w5s.googlevideo.com
127.0.0.1  rr3---sn-gvnuxaxjvh-3w5s.googlevideo.com
```

По необходимости включаю **adguard-vpn** вот так:
```
adguardvpn-cli connect
```
и выключаю, чтобы попасть в лимит трафика:
```
adguardvpn-cli disconnect
```


### Ссылки

Linux, Debian 13.

* [ByeDPI (ciadpi)](https://github.com/hufrea/byedpi/releases/tag/v0.17.3)
* [AdGuard VPN](https://github.com/AdguardTeam/AdGuardVPNCLI/releases/tag/v1.7.12-release)
* [Proxy SwitchyOmega 3 (ZeroOmega)](https://chromewebstore.google.com/detail/proxy-switchyomega-3-zero/pfnededegaaopdmhkdmcofjmoldfiped)

**Comment by lynrayy** [comment]
Есть новая актуальная строка?

В твиттер не могу зайти, из-за впна не пускает

**Comment by SevaOv** [comment]
Всем доброго времени суток! 
Поделитесь пожалуйста аргументом командной строки для Discord или может быть для Telegram. 
Для Ютуба у меня есть:
  -d1 -d3+s -s6+s -d9+s -s12+s -d15+s -s20+s -d25+s -s30+s -d35+s -r1+s -S -a1 -As -d1 -d3+s -s6+s -d9+s -s12+s -d15+s -s20+s -d25+s -s30+s -d35+s -S -a1
(работает хорошо на 30.05.26)

**Reply by DroidProger** [reply]
Про телегу через byedpi можно забыть - на проводном билайне не проходит даже пинг на t.me. Лучше используй встроеный прокси mtproto, благо в самой телеге есть каналы в которых эти прокси регулярно выкладывают, правда они и отмирают быстро, поэтому надо постоянно добавлять новые и удалять старые.

**Reply by yaroslaff** [reply]
а у меня на МТС (мобильный) телега взлетела через byebyedpi с этой стратегией.

**Reply by DroidProger** [reply]
Попробовал. На пк телега вроде заработала. На смартфоне нет

**Reply by denizzzka** [reply]
Телегу (напрямую, без проксей), скорее всего, банят просто по диапазону её IP-шников. Поэтому, обход через обман DPI не факт что сработает в принципе

**Reply by SevaOv** [reply]
Для ТГ на ПК я нашел приложуху с хорошими прокси, всез перебоев почти работает: https://github.com/Flowseal/tg-ws-proxy/releases
Отзывы хорошие у него, на вирусы проверял, все чисто!

**Comment by DroidProger** [comment]
К сожалению телега не работает, (ранее я писал, что заработала, но потом увидел, что она шла через впн а не ди-пи-ай) пинг не идет, traceroute не показывает узлы. Билайн, провод, юг

**Comment by pidrilaebanaja** [comment]
если обновлений не будет, то эту дожмут так же как и гудбайдпи в своё время, потом возьмутся за zapret

**Comment by hagnozo** [comment]
МГТС  сегодня с утра  
`ciadpi -o1 -a1 -r-5+se -H:"ggpht.com googlevideo.com youtube.com youtu.be ytimg.com"`

Для билайн советуют это 
`-H:"googleapis.com" -s1 -r1+s -An -H:"ggpht.com googlevideo.com youtube.com youtu.be ytimg.com" -o1 -r-5+se`

Гугл-переводчик усоп?

**Comment by Davidbest404** [comment]
Тут ни у кого нету tele2 оператора?
😢

**Reply by DroidProger** [reply]
Пробуй параметры как для ростелекома

**Comment by SevaOv** [comment]
Прокси для ТГ, как впн включается. Работает хорошо на Т2
https://github.com/Flowseal/tg-ws-proxy/releases

**Comment by denizzzka** [comment]
~~Так, они что-то поменяли? Домен googlevideo.com теперь совсем не работает, ни с какими настройками?~~

У нас теперь выкусывают адреса DNS ютуба из ответов серверов 8.8.8.8 и 8.8.4.4

1.1.1.1 и 9.9.9.9 пока работают

**Reply by Mihrutkin** [reply]
А пробовали прогонять тест byebyepdi
-o1 -a1 -r-5+se
У меня даже с их дефолтными на ура

**Reply by denizzzka** [reply]
Сменил ДНС и всё стало снова работать с указанными здесь на странице настройками

**Reply by SevaOv** [reply]
Попробовал как все сказал, но с моим МГТС это все не работает. Даже коды менял с днс, но все бестолку

**Reply by Mihrutkin** [reply]
> Попробовал как все сказал, но с моим МГТС это все не работает. Даже коды менял с днс, но все бестолку

Переходи на zapret

**Reply by denizzzka** [reply]
Они решили начать фильтрацию DNS чтобы разгрузить DPI-фильтры

DNS over HTTPS (DoH) работает пока ещё

**Reply by SevaOv** [reply]
Запрет только на ПК, мне на телефон надо. А то как Ютубчик смотреть за едой?

**Comment by SevaOv** [comment]
Аргумент для командной строки:
-s0 -o1 -d1 -r1+s -Ar -o1 -At -f-1 -r1+s -As

Во вкладке DNS ввести значение 8.8.8.8

На Т2 хорошо работает

**Comment by Mihrutkin** [comment]
Ты ошибаешься:
https://github.com/CherretGit/zaprett-app

Также можно для всех домашних сделать через роутер на все устройства в доме

**Reply by denizzzka** [reply]
Прокси удобнее ядерных модулей

**Comment by Mihrutkin** [comment]
Нет.
Podkop+warp+zapre/byedpi
У тебя дома интернет просто начинает работать бесшовно

**Reply by denizzzka** [reply]
Podkop+warp+zapre/byedpi
@
просто
